video/out/cocoa_common.m

```diff
@@ -873,4 +883,16 @@ - (void)windowDidBecomeKey:(NSNotification *)notification
     [self didChangeMousePosition];
 }
 
+- (void)windowDidMiniaturize:(NSNotification *)notification
+{
+    struct vo_cocoa_state *s = self.vout->cocoa;
+    s->pending_events |= VO_EVENT_WIN_STATE;
+}
+
+- (void)windowDidDeminiaturize:(NSNotification *)notification
```

Comment:
It's been merged so the point is moot, but unless I've missed something, __`MpvCocoaAdapter`__ does not implement the NSWindowDelegate protocol. __`MpvVideoWindow`__ is the class that is the WindowDelegate, in `window.h`. Since `MpvCocoaAdapter` only inherits from NSObject, it doesn't get these methods declared from the NSWindowDelegate protocol. All method calls are sent as messages in obj-c anyway, so having them declared in the interface isn't necessary for them to work, but if `MpvCocoaAdapter` were the WindowDelegate you wouldn't need to curry these event calls to it from `MpvVideoWindow`.

Well anyway, I'm just trying to make sure I didn't misunderstand the code structure.